### ISSUE TITLE
The newest Windows update restricts access to the supabase default ports

### ISSUE THREAD

**Issue by devdev-dev** [supabase/cli#189]
# Bug report

## Describe the bug

With the newest windows update the default ports used by the init script are blocked by Windows.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Get the latest Windows Updates
2. Run "netsh interface ipv4 show excludedportrange protocol=tcp" in a cmd (as described [here](https://www.deviantdev.com/journal/docker-ports-socket-forbidden))
3. Check the list of restricted ports
4. The port range used by windows is part of the exclusion range

## Expected behavior

You should change the ports respecting the new exclusion range.

## Screenshots

![image](https://user-images.githubusercontent.com/1813071/157971332-3125cd12-f16b-495d-979e-424eebddd656.png)

## System information

- OS: Windows
- Supabase CLI 0.17.3

**Comment by sweatybridge** [comment]
This seems like a problem with Hyper-V reserving random dynamic port ranges https://stackoverflow.com/a/71190107. For example, the reserved port range on my machine doesn't conflict with CLI ports.

```PS
PS C:\Users\qiao> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.19041.1682
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.1682
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1


PS C:\Users\qiao> netsh interface ipv4 show excludedportrange protocol=tcp

Protocol tcp Port Exclusion Ranges

Start Port    End Port
----------    --------
      9001        9001
     50000       50059     *

* - Administered port exclusions.
```

Since this issue is machine specific, it's best to follow the suggestion in the SO post to instruct Windows to modify this range. I will close this issue as won't fix.
